src/main/java/com/citihub/configr/namespace/NamespaceService.java

```diff
@@ -78,6 +92,45 @@ public Namespace storeNamespace(Map<String, Object> json, String path, boolean s
     }
   }
 
+  private void validateNamespace(Namespace namespace)
+      throws SchemaValidationException, JsonProcessingException {
+
+    // TODO: Should we use metadataService.getMetadataForNamespace ?
+    // or will namespace.getMetadata() perform lazy loading?
```

Comment:
not at the moment no - the "Namespace" object is the logical DTO but the storage model is broken up entirely